app/handlers/sessions_create.rb

```diff
@@ -120,23 +120,36 @@ def handle_during_signup
   end
 
   def handle_while_logged_in
-    # Normally this happens when adding authentications on the profile screen.
+    # Attempt to login when already logged in
 
+    # Same user: don't do anything
     return :no_action if authentication_user == current_user
```

Comment:
Can this be moved out to the main `handle`?

Reply:
I think it makes more sense here because current_user will be nil elsewhere?